src/kOS/Function/FunctionBase.cs

```diff
@@ -35,7 +35,7 @@ public object ReturnValue
                 internalReturn = value;
             }
         }
-        private object internalReturn = null;
+        private object internalReturn = 0; // really should be 'null', but kerboscript can't deal with that.
```

Comment:
I think we need to make a decision about how to handle empty returns.  In some instances, we return "null" which the users can't deal with.  Setting the return to 0 doesn't really help them any better (is the return value 0 because it's really 0, or because there is no return?) but at least it provides an object.  If the return value is to be left as 0 in FunctionBase, I think that we should have line 139 of DelegateSuffixResult match.  I'll make that change, unless some one disagrees.